edgelet/edgelet-kube/src/runtime.rs

```diff
@@ -58,7 +64,10 @@ pub trait KubeRuntimeData {
     fn management_uri(&self) -> &Url;
 }
 
-impl<S> KubeRuntimeData for KubeModuleRuntime<S> {
+impl<T, S> KubeRuntimeData for KubeModuleRuntime<T, S>
+where
+    T: Clone,
```

Comment:
Same here. Shouldn't need to bound `T` with `Clone`.

Reply:
Ok